tests/unit/files/non_utf8_metadata_unsigned.xml

```diff
@@ -0,0 +1,14 @@
+<md:EntityDescriptor xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata" entityID="https://accounts.google.com/o/saml2?idpid=C03oypjtr" validUntil="2028-04-02T06:47:09.000Z">
```

Comment:
I think the `<?xml version="1.0" encoding="UTF-8"?>` at the begin of the xml file is necessary for this test?

Reply:
This is the problematic metadata, as provider by Google, so I'd rather keep it exactly as it is

Reply:
I think this file is the sample that goes into the unit test. Does it mean that we need to keep the problematic metadata so we can make sure the fix works?

Reply:
That was the idea, yes